src/main/java/org/betonquest/betonquest/compatibility/protocollib/wrappers/WrapperPlayServerEntityMetadata.java

```diff
@@ -0,0 +1,73 @@
+package org.betonquest.betonquest.compatibility.protocollib.wrappers;
+
+import com.comphenix.protocol.PacketType;
+import com.comphenix.protocol.wrappers.WrappedWatchableObject;
+import org.bukkit.World;
+import org.bukkit.entity.Entity;
+
+import java.util.List;
+
+@SuppressWarnings("PMD.CommentRequired")
+public class WrapperPlayServerEntityMetadata extends PacketHandlerDecorator{
```

Comment:
This class is already implemented in the dependency PacketWrapper. There should be no need to copy paste this class into BetonQuest